### ISSUE TITLE
[x] Data export: Issues with various formats

### ISSUE THREAD

**Issue by amberreeves** [InteractiveMechanics/CollegeWomen_Drupal7#74]
From the Home page I entered "vassar" in the search box.
I then clicked each export format to download my results set. 
Here is what I saw:

**CSV** only one column header/field label displays, article_year. i don't see the column labels for the title, URL, date, etc.
**example:** https://drive.google.com/file/d/1P2EGDteHO-LpMAY8_BZC6z4nOwUBwafZ/view?usp=sharing

**JSON** the JSON data contained virtually no data except format and article_year.  The format field is represented by "no name" in the data, which means that the data came through but the label didn't. (JSON data is kind of a fancy associative array, meaning that the name/value pairs in the data, like "creator = bromley, frances" should be represented by something like [ {"creator":"bromley, frances"} ]
**example**: https://drive.google.com/file/d/1rGVOPKEFNRxrI2ish3a5co-Yk1pb9l5D/view?usp=sharing

**TXT** the text is just odd. I don't see how it's necessarily delimited, but I think it's trying to do: [field label] | carriage return | data. Since the field labels aren't coming through, it's a little messy, but the data *is* coming through.
**example:** https://drive.google.com/file/d/1_E1JvtdndL5L4Z5ibgYCVAjeZGVnD1Cg/view?usp=sharing

In summary, it seems like:
- Exporting the field *values* works for CSV and TXT but not for JSON.
- No field names are coming through *except* article_date (which isn't actually represented in the search results)

**Comment by mtedeschi** [comment]
Updated on production. Notes on what changed:

CSV now has field labels.
TXT now has field labels.
JSON is now showing date, with appropriate labels.

**Comment by amberreeves** [comment]
looks good
